.github/workflows/release.yaml

```
@@ -0,0 +1,147 @@
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Checkout BMF repository (specific branch)
+      run: |
+        sudo apt update
+        sudo apt install -y make git pkg-config libssl-dev cmake binutils-dev libgoogle-glog-dev gcc g++ golang wget libgl1
+        sudo apt install -y python3.9 python3-dev python3-pip libsndfile1 libsndfile1-dev
+        # sudo python3 -m pip install timeout_decorator numpy onnxruntime pytest opencv-python librosa
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)